pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiTextQuestion.java

```diff
@@ -31,11 +33,13 @@ int insert(@Bind("questionId") long questionId,
             + "INNER JOIN suggestion_type AS s_type ON s_type.suggestion_type_code = :suggestionType "
             + "SET tq.input_type_id = i_type.text_question_input_type_id, "
             + "     tq.suggestion_type_id = s_type.suggestion_type_id, "
-            + "     tq.placeholder_template_id = :placeholderTemplateId "
+            + "     tq.placeholder_template_id = :placeholderTemplateId, "
+            + "     tq.confirm_placeholder_template_id = :confirmPlaceholderTemplateId "
             + "WHERE tq.question_id = :questionId")
     boolean update(@Bind("questionId") long questionId,
                    @Bind("inputType") TextInputType inputType,
                    @Bind("suggestionType") SuggestionType suggestionType,
-                   @Bind("placeholderTemplateId") Long placeholderTemplateId);
+                   @Bind("placeholderTemplateId") Long placeholderTemplateId,
+                   @Bind("confirmPlaceholderTemplateId") Long confirmPlaceholderTemplateId);
```

Comment:
Confirm placeholder goes along with confirmEntry, confirmPrompt, and mismatchMessage. Does it make sense to allow updating `confirmPlaceholderTemplateId` when  the other "confirm" properties are not part of the method? I suggest removing `confirmPlaceholderTemplateId` from here.

Reply:
Removed